kratos/python_scripts/set_initial_state_process.py

```diff
@@ -1,51 +1,62 @@
-# Importing the Kratos Library
+"""Process for setting an initial state (strain, stress, deformation radient)."""
 import KratosMultiphysics
 
 
 def Factory(settings, Model):
+    """Create process."""
     if not isinstance(settings, KratosMultiphysics.Parameters):
         raise Exception(
             "expected input shall be a Parameters object, encapsulating a json string"
         )
     return SetInitialStateProcess(Model, settings["Parameters"])
 
 
-# All the processes python processes should be derived from "Process"
+def param_to_str(param):
+    """
+    Convert input to string type.
 
+    Keyword arguments:
+    param -- Accepted types are Parameters double, Parameters string, int, float, str.
+    Returns:
+    string expression of the input
+    """
+    if isinstance(param, KratosMultiphysics.Parameters):
+        if param.IsNumber():
+            sparam = str(param.GetDouble())
+        elif param.IsString():
+            sparam = param.GetString()
+    elif isinstance(param, (int, float, str)):
+        sparam = str(param)
+    else:
+        msg = "SetInitialStateProcess: Value must be scalar or string"
+        raise Exception(msg)
+    return sparam
+
+
+def str_to_function(expr):
```

Comment:
```suggestion
def __StrToFunction(expr):
```
[Style guide](https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide). I'd also use name mangling with this one.

Reply:
Done